acsoo/templates/project/+project.name+/deploy-test.bob

```diff
@@ -1,8 +1,15 @@
-#!/usr/bin/env python
+{{% if odoo.series in ['11.0'] %}}
```

Comment:
So here it could be based on the python version set in the pre_render instead of odoo series.